Evergreen/MainWindow/MainWindowController.swift

```diff
@@ -89,6 +97,13 @@ class MainWindowController : NSWindowController, NSUserInterfaceValidations {
 			openInBrowser(link)
 		}		
 	}
+    
+    @IBAction func showShareWindow(_ sender: AnyObject) {
+        if let link = currentLink, let url = URL(string: link) {
+            let picker = NSSharingServicePicker(items: [url])
+            picker.show(relativeTo: sender.bounds, of: sender as! NSView, preferredEdge: NSRectEdge.minY)
+        }
```

Comment:
You're absolutely right, I totally missed this somehow, thanks for taking a look! Cleaned this up and switched to using a `guard` as well.